src/cs/helpers.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  AnyType,
+  Kind,
+  List,
+  Map,
+  Named,
+  Optional,
+} from "@apexlang/core/model";
+import { pascalCase } from "../utils";
+import { translations } from "./constant";
+import * as crypto from "crypto";
```

Comment:
what is `crypto` being used for? Its causing a build error.

```
src/cs/helpers.ts(11,25): error TS2307: Cannot find module 'crypto' or its corresponding type declarations.
```

If you really need it, you'll need to add it to package.json.